src/common/bitstream_test.cpp

```diff
@@ -119,6 +121,88 @@ void testBitstreamReadBits() {
     }
 }
 
+void testBitstreamReadQuantitizedFloat() {
+    static std::vector<uint8_t> expectedBuf = std::vector<uint8_t>({
+        0x6C, 0x2D, 0x76, 0x55, 0x35, 0xCA, 0x16 //6C2D7 65535 CA16
+    });
+    BitStream bitstream(expectedBuf);
+
+    float x;
+    float y;
+    float z;
+    bitstream.readQuantitizedFloat(x, 20, 8192.0f);
+    bitstream.readQuantitizedFloat(y, 20, 8192.0f);
+    bitstream.readQuantitizedFloat(z, 16, 1024.0f);
+    assertEqual( compareFloats(x, 3674.85f, 100), 0);
+    assertEqual( compareFloats(y, 2726.79f, 100), 0);
+    assertEqual( compareFloats(z, 91.1576f, 100), 0);
+}
+
+void testBitstreamWriteQuantitizedFloat() {
```

Comment:
The main issue I had going into the write test was precision.  I took the same values I had calculated for the read test, tried to use them for the write test, only to discover that the the previous's test wasn't displaying the full decimal precision during an assert failure.  That output was two places after the decimal and then rounded, whereas the actual number was four or so, unrounded.

For this write test, using hard numbers, the x-value was fine, but the y-value was off by 2, and the z-value was off by 1, iirc, a matter of some thousandths digit.